puppet/manifests/default.pp

```diff
@@ -12,6 +32,35 @@
   include seed_stack::load_balancer
 
   include docker_registry
+
+  include glusterfs_common
+
+  gluster_peer { 'standalone.seed-stack.local':
+    require => Package['glusterfs-server'],
```

Comment:
That's from when we were using the other module, where we do need the require in puppetland. It doesn't hurt, but the only effect it'll have is to emit a different error if the package isn't installed.